lib/i18n/backend/base.rb

```diff
@@ -236,14 +236,29 @@ def load_rb(filename)
           eval(IO.read(filename), binding, filename)
         end
 
+      YAML_LOAD_OPTIONS = {
+        symbolize_names: true,
+        freeze: true,
+      }.freeze
+
+      # Feature tests YAML load options and only selects those that
+      # are supported in the current version of Pysch.
+      def yaml_load_options
+        @yaml_load_options ||= YAML_LOAD_OPTIONS.select do |k, v|
```

Comment:
Also you don't need a method to memoize this, you can directly define `YAML_LOAD_OPTIONS` with a dynamic select as YAML is required at the top of the file anyway.